broker.go

```diff
@@ -402,6 +402,24 @@ func (b *Broker) Bind(instanceID, bindingID string, details brokerapi.BindDetail
 		return binding, err
 	}
 
+	if m, ok := creds.(map[string]interface{}); ok {
+		if ok {
+			elem, ok := m["admin_username"]
+			if elem, ok = m["admin_username"]; ok {
+				if ok {
+					delete(m, "admin_username")
+				}
+			}
+			if elem, ok = m["admin_password"]; ok {
+				if ok {
+					delete(m, "admin_password")
+				}
+			}
+			_ = elem
+			creds = m
```

Comment:
shouldn't be needed, when you delete you are already operating on the pointer to `creds`

Reply:
cleaner version